.gitignore

```diff
@@ -11,3 +11,6 @@ lerna-debug.log
 .idea/
 .vscode/
 *.orig
+/test-results/
```

Comment:
Ignore the playwright stuff being added to .gitignore
This will be relevant once the playwright setup is in place